src/udev/udevd.c

```diff
@@ -1055,6 +1103,8 @@ static int on_uevent(sd_device_monitor *monitor, sd_device *dev, void *userdata)
                 return 1;
         }
 
+        (void) event_queue_assume_block_device_unlocked(manager, dev);
```

Comment:
(doesn't really matter, but we could in theory speed up the inotify case, and immediately unblock the uevents for a device when we see inotify, and not wait until the retrigger uevent percolated through the kernel back to us, no?)